code/modules/mob/living/carbon/human/life/handle_stasis_bag.dm

```diff
@@ -11,3 +11,12 @@
 		if(STASIS_IN_CRYO_CELL)
 			if(sleeping < 10)
 				sleeping += 10 //Puts the mob to sleep indefinitely.
+
+/mob/living/carbon/human/proc/handle_stasis_bag_with_APS()
+	for(var/datum/reagent/antiparasitic_reagent in src.reagents.reagent_list)
+		var/datum/chem_property/property = antiparasitic_reagent.get_property(PROPERTY_ANTIPARASITIC) //Adrenaline helps greatly at restarting the heart
+		if(property)
+			if(prob(25))
+				to_chat(src, SPAN_NOTICE("A strange sensation of anxiety sweeps over you as something inside you relaxes under the stasis field.")) //Inform the marine the stasis field is killing
```

Comment:
dunno about this but you are not supposed to be able to know when you are early infected